code/game/objects/items/rogueweapons/melee/blunt.dm

```diff
@@ -3,12 +3,11 @@
 /datum/intent/mace/strike
 	name = "strike"
 	blade_class = BCLASS_BLUNT
-	attack_verb = list("strikes", "hits")
+	attack_verb = list("strikes", "whacks")
 	hitsound = list('sound/combat/hits/blunt/metalblunt (1).ogg', 'sound/combat/hits/blunt/metalblunt (2).ogg', 'sound/combat/hits/blunt/metalblunt (3).ogg')
 	chargetime = 0
-	penfactor = BLUNT_DEFAULT_PENFACTOR
-	damfactor = 1.1
-	swingdelay = 0
+	penfactor = 20
+	swingdelay = 4 // Idea behind it, is to balance out swift intent and give this weapon a purpose more damage at the cost of delay.
```

Comment:
do not do this. this will make all maces terrible. 
swingdelay means if your opponent is not standing directly next to you in the case of 1 tile range attacks, it will not hit. this means that if you use a mace and your opponent moves away from you, it is impossible to hit them. 
blunt attacks are already worse by default due to not causing any wounds on a normal attack, this much is extremely unnecessary.